ukrainian/break-through-wall/try-the-taste/try-the-taste.md

```diff
@@ -1 +1,9 @@
-Зефір виявився найсмачнішим, що Ви коштували в своєму житті!
\ No newline at end of file
+Зефір виявився найсмачнішим, що Ви коштували в своєму житті!
+
+Вам дуже закортіло набрати зефіру у кишені, та виделка залишилась у кімнаті, діра у яку майже зовсім затягнулась.
+
+Що будете робити?
+
+[Повернутися до кімнати за виделкою](../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.